packages/fetchai/skills/simple_data_request/behaviours.py

```diff
@@ -62,6 +63,13 @@ def act(self) -> None:
 
         :return: None
         """
+        if self.lookup_termination_key is not None:
```

Comment:
a skill can be deactivated via is_active flag on context, use it here

Reply:
The `self.lookup_termination_key` in `shared_state` may not be set at this point by the other skill yet (because it hasn't finished yet) but could be set later. So can't deactivate this skill if the condition fails.